app/controllers/partners_controller.rb

```diff
@@ -11,6 +11,7 @@ def new
   def create
     @partner = Partner.new(partner_params)
     authorize @partner
+    @partner.statement_accepted_at if @partner.statement
```

Comment:
Je suppose qu'il faut rajouter ici `= Time.now` aussi on m'a toujours recommandé d'utiliser Time.zone.now afin de prendre en compte la time zone pour le date time.

```suggestion
    @partner.statement_accepted_at = Time.zone.now if @partner.statement
```